src/Hl7.Fhir.Shims.Base/Specification/Source/SnapshotSource.cs

```diff
@@ -96,7 +98,9 @@ private async Tasks.Task<Resource> ensureSnapshot(Resource res)
         {
             if (res is StructureDefinition sd)
             {
-                if (!sd.HasSnapshot || Generator.Settings.ForceRegenerateSnapshots || !sd.Snapshot.IsCreatedBySnapshotGenerator())
+#pragma warning disable CS0618 // Type or member is obsolete
+                if (!sd.HasSnapshot || (Generator.Settings.ForceRegenerateSnapshots && !sd.Snapshot.IsCreatedBySnapshotGenerator()))
```

Comment:
As Ewout says in https://github.com/FirelyTeam/firely-net-sdk/issues/2802 this will surely result in regressions. We shouldn't change this.